docs/Userguide_login.rst

```diff
@@ -89,25 +225,90 @@ automatically perform some of the below steps for you. You can install it with
     ``mila init`` in order to apply new features or bug fixes.
 
 
-SSH Config
-----------
+.. manual_ssh_config:
+
+Manual SSH configuration
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you would like to set entries in your ``~/.ssh/config`` file manually for
+advanced use-cases, you may use the following as inspiration:
+
+.. code-block:: text
+
+    #   Mila
+    Host mila             login.server.mila.quebec
+        Hostname          login.server.mila.quebec
+    Host mila1    login-1.login.server.mila.quebec
+        Hostname  login-1.login.server.mila.quebec
+    Host mila2    login-2.login.server.mila.quebec
+        Hostname  login-2.login.server.mila.quebec
+    Host mila3    login-3.login.server.mila.quebec
+        Hostname  login-3.login.server.mila.quebec
+    Host mila4    login-4.login.server.mila.quebec
+        Hostname  login-4.login.server.mila.quebec
+    Host mila5    login-5.login.server.mila.quebec
+        Hostname  login-5.login.server.mila.quebec
```

Comment:
Not related to this PR, but I wonder if we should simplify to `login.mila.quebec` and `loginX.mila.quebec` where `X is in  [1-5]`? (We would obviously keep current DNS records, but using simpler and shorter DNS names would be better going forward.)

Reply:
Perhaps. I would have to study what is possible with `ssh_config(5)`'s `CanonicalizePermittedCNAMEs` and all the rest of the `Canonicalize*` jazz.

Reply:
I'm not talking of using `Canonicalize*`. I only wonder if we could shorten the DNS name of the login nodes.

This block would be:
```
    Host mila      login.mila.quebec
        Hostname   login.mila.quebec
    Host mila1    login1.mila.quebec
        Hostname  login1.mila.quebec
    Host mila2    login2.mila.quebec
        Hostname  login2.mila.quebec
    Host mila3    login3.mila.quebec
        Hostname  login3.mila.quebec
    Host mila4    login4.mila.quebec
        Hostname  login4.mila.quebec
    Host mila5    login5.mila.quebec
        Hostname  login5.mila.quebec
```

Reply:
I would not be opposed to shorter names, but I do see *some* value in keeping the 3rd level of DNS clear by the use of the `server.` segment, making most subdomains available for other purposes.